apps/main/src/dex/entities/appstats-tvl.ts

```diff
@@ -0,0 +1,18 @@
+import type { CurveApi } from '@/dex/types/main.types'
+import { queryFactory } from '@ui-kit/lib/model/query'
+import { EmptyValidationSuite } from '@ui-kit/lib'
+import curvejsApi from '@/dex/lib/curvejs'
+
+async function _fetchAppStatsTvl({ curve }: { curve: CurveApi | null }) {
+  if (!curve) return null
```

Comment:
Let's move curve checks to a validation suite like in crvUSD:
https://github.com/curvefi/curve-frontend/blob/67eaef126b3afc79bb039fc227ec49c4d925798f/apps/main/src/lend/entities/chain/validation.ts#L19-L20